src/site/xdoc/server/config-blobstore.xml

```diff
@@ -56,6 +56,14 @@
                 </dl>
             </subsection>
 
+            <subsection name="Hybrid BlobStore option to always write to ObjectStorage">
+                <dl>
+                    <dt><strong>hybrid.duplicate.writes</strong></dt>
+                    <dd>DEFAULT: false, must be true/false. It happen only 'value = true' and it stored in cassandra.</dd>
+                    <dd>Enabling this option in the only way to make the adoption of hybrid blob store reversible, however you will loose performance/cost gains on writes</dd>
```

Comment:
You mean a trade-off in performance? If yes then s/loose/lose.

Reply:
Yes, typing mistake.